### ISSUE TITLE
Add support of the PyTorch 0.3

### ISSUE THREAD

**Issue by nerox8664** [nerox8664/pytorch2keras#6]
There are some problems with new PyTorch release:

1. The current traverse function is no longer accurate.
2. Number of layers was renamed.
3. Difference in the arguments / calculation.

What to do:

- [ ]  Implement new traverse function (or update / modify existent)
- [ ]  Make changes in the layers:
  - [x]  Conv2d 
  - [ ]  BatchNorm2d 
  - [ ]  Linear
  - [ ]  Pooling
  - [ ]  Activations
  - [ ]  Reshape / View
  - [ ]  ConvTranspose2d
  - [ ]  Dropout
  - [ ]  Element-wise operations
  - [ ]  Concatenation

**Comment by nerox8664** [comment]
Updates:

1. The new library (ATEN) is used in the PyTorch for pooling and some other layers. It cause that in the some Backward objects doesn't storing any attributes (kernel_size for pooling and etc).
2. There is some renames in the grad_fn.

So, as I can see, a good decision for now - [wait for that moment.](https://github.com/pytorch/pytorch/issues/4051)

**Comment by nerox8664** [comment]
0.3 version will not supported. I'm switching to the 0.4 with full ATen support.
